packages/jsActions/nanoflow-actions-web/src/ToggleSidebar.ts

```diff
@@ -1,22 +0,0 @@
-// This file was generated by Mendix Studio Pro.
```

Comment:
Why is this _in this PR_? Couldn't find anything in the original comment and it feels unrelated

Reply:
I edited the PR description. Next time I will create a separate PR for such things.